frontend/fingertips-frontend/components/pages/chart/chart.test.tsx

```diff
@@ -0,0 +1,39 @@
+import { render, screen } from '@testing-library/react';
+import { Chart } from '@/components/pages/chart/index';
+import { expect } from '@jest/globals';
+import { WeatherForecast } from '@/generated-sources/api-client';
+
+const mockData: WeatherForecast[] = [
+  {
+    date: new Date('2024-11-01T00:00:00.000Z'),
+    temperatureC: -30,
+    temperatureF: -21,
+    summary: 'Freezing',
+  },
+  {
+    date: new Date('2024-11-01T00:00:00.000Z'),
+    temperatureC: 0,
+    temperatureF: 32,
+    summary: 'Bracing',
+  },
+];
+
+test('should render the Highcharts react component', () => {
+  render(<Chart data={mockData} />);
+  const highcharts = screen.getByTestId('highcharts-react-component');
+  expect(highcharts).toBeInTheDocument();
+});
+
+test('should render the LineChart component with the correct props', () => {
+  render(<Chart data={mockData} />);
+
+  const lineChartTitle = screen.getByText('Line Chart');
+  expect(lineChartTitle).toBeInTheDocument();
+});
```

Comment:
This is just checking that the title has been rendered, so the test name probably needs updating to reflect that.